src/kibana/components/config/defaults.js

```diff
@@ -43,6 +43,21 @@ define(function (require) {
       value: 6,
       description: 'The maximum geoHash size allowed in a tile map',
     },
+    'visualization:tileMap:wms': {
+      value: false,
+      description: 'Set for a WMS mapping provider, see <a href="http://leafletjs.com/reference.html#tilelayer-wms">leafletjs reference</a>'
```

Comment:
The link here indicates this controls a function that takes an object, but the value is a boolean? I think I understand from the code what this is supposed todo, but I don't think other users will.

Also note that all links should target an empty window to avoid redirecting the user out of the application when they click them.